subsys/bluetooth/mesh/vnd/le_pair_resp.c

```diff
@@ -92,7 +93,15 @@ static int bt_mesh_le_pair_resp_init(const struct bt_mesh_model *model)
 
 static void bt_mesh_le_pair_resp_reset(const struct bt_mesh_model *model)
 {
+	int err;
+
 	bt_mesh_le_pair_resp_passkey_invalidate();
+
+	/* Clear all bonding information when receiving node reset */
+	err = bt_unpair(BT_ID_DEFAULT, NULL);
+	if (err) {
+		LOG_ERR("Failed to clear bonds during reset (err: %d)", err);
```

Comment:
```suggestion
		LOG_ERR("Unpair err: %d", err);
```

It is possible to make log string not so large compared to amount of hex code added by function call, without loss of meaning.